Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -94,38 +94,49 @@ bool AVIDump::Start(int w, int h)
 
 bool AVIDump::CreateVideoFile()
 {
-  const char s_format[] = "avi";
-
-  AVCodec* codec = nullptr;
-
-  std::stringstream file_ss;
-  file_ss << File::GetUserPath(D_DUMPFRAMES_IDX)
-                   << "framedump" << s_file_index
-                   << "." << s_format;
-  std::string filename = file_ss.str();
-  File::CreateFullPath(filename);
-
-  // Ask to delete file
-  if (File::Exists(filename))
+  const std::string& s_format = g_Config.sDumpFormat;
```

Comment:
Yeah, since g_Config is a global (ugh), this is alright.

As an aside, local const references do have a peculiarity about them that non-const references don't. Since local const references (but *not* non-const references) lengthen the lifetime of a temporary to the lifetime of the const reference itself, the following code is perfectly valid.

```cpp
void SomeFunction()
{
  const int& ref = 2;
  // More code would follow...
}
```

which would effectively be equivalent to:

```cpp
void SomeFunction()
{
  const int some_val = 2;
  const int& ref = some_val;
}